packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -83,17 +83,32 @@ class _DefaultDoctorValidatorsProvider implements DoctorValidatorsProvider {
       if (globals.iosWorkflow.appliesToHostPlatform || macOSWorkflow.appliesToHostPlatform)
         GroupedValidator(<DoctorValidator>[XcodeValidator(xcode: globals.xcode, userMessages: userMessages), cocoapodsValidator]),
       if (webWorkflow.appliesToHostPlatform)
-        ChromeValidator(
-          chromiumLauncher: ChromiumLauncher(
-            browserFinder: findChromeExecutable,
-            fileSystem: globals.fs,
-            logger: globals.logger,
-            operatingSystemUtils: globals.os,
-            platform:  globals.platform,
-            processManager: globals.processManager,
+        GroupedValidator(<DoctorValidator>[
```

Comment:
I'm a bit concerned about updating the doctor check. Maybe start small with just -d edge